corehq/apps/app_manager/templates/app_manager/form_designer.html

```diff
@@ -19,6 +19,13 @@
             plugins: {{ plugins|JSON }},
             features: {{ features|JSON }},
             core: {
+                dataSources: [
+                    {
+                        key: 'fixture',
+                        name: 'Fixtures',
```

Comment:
This name isn't actually used for anything right now. Reused it from an old branch. Could delete that.

The only names that are used are the ones for the dropdown/autocomplete, so it would only be something like the fixture name.